language/move-prover/src/bytecode_translator.rs

```diff
@@ -1371,6 +1371,17 @@ impl<'env> ModuleTranslator<'env> {
                 );
                 emitln!(self.writer, &update_and_track_local(*dest, "$tmp"));
             }
+            // Use a dummy call for unimplemented instructions Shl and Shr
+            Binary(BinaryOp::Shl, dest, op1, op2) | Binary(BinaryOp::Shr, dest, op1, op2) => {
```

Comment:
I did took care of this in my pending #3391. Instead of the above, I introduced stubs for Shl and Shr in the library. We can race who lands earlier.

Reply:
OK looks like you win.  That works too.